test/prism/snapshots_test.rb

```diff
@@ -60,7 +60,7 @@ def assert_snapshot(fixture)
         # If the snapshot file exists, then assert that the printed value
         # matches the snapshot.
         assert_equal(saved, printed)
-      else
+      elsif !Fixture.custom_base_path?
```

Comment:
Can you change this to just skip this whole test if there's a custom base path? I found it confusing to have this case in here.

Something like `return if Fixture.custom_base_path?` at the top after the test helper require would probably work.

Reply:
Agreed, that's cleaner